src/index.ts

```diff
@@ -60,58 +65,63 @@ async function run(): Promise<void> {
         }
         else {
             core.warning(`The version of ${imageToPush} in the Podman image storage is more recent `
-                + `than the version in the Docker image storage. The image from the Podman image `
+                + `than the version in the Docker image storage. Tag(s) of the image from the Podman image `
                 + `storage will be pushed.`);
         }
     }
     else if (isPresentInDocker) {
         imageToPush = `${dockerBaseUrl}/${imageToPush}`;
         core.info(`${imageToPush} was found in the Docker image storage, but not in the Podman `
-            + `image storage. The image will be pulled into Podman image storage, pushed, and then `
+            + `image storage. Tag(s) of the image will be pulled into Podman image storage, pushed, and then `
             + `removed from the Podman image storage.`);
         isImageFromDocker = true;
     }
 
-    let pushMsg = `Pushing ${imageToPush} to ${registry}`;
+    let pushMsg = `Pushing ${imageToPush} with tags ${tagsList.toString()} to ${registry}`;
```

Comment:
You can get nicer looking stringified arrays with `tagsList.join(", ")` for example.